events/jsonschema/io.mintmetrics.mojito/mojito_failure/jsonschema/1-0-0

```diff
@@ -1,5 +1,6 @@
 {
   "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description" : "In complex builds, it's hard to avoid errors firing from experiment variants (such as obscure browsers, products with unusual features or deployments breaking tests mid-way through). Errors can mean the difference between your treatment working or breaking.",
```

Comment:
Following the Snowplow example of developing schemas [1], I think we can simplify this a little. E.g.:

```Schema for a Mojito failure event```

1: https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow.badrows/enrichment_failures/jsonschema/2-0-0

Reply:
Sure, following Snowplow here makes sense. I was never certain, if these short descriptions in the Snowplow docs were maybe just hastily written down examples. Your link to the badrow schema seems more like there is a pattern to follow. :-) I will update the merge request.